src/core/server/tools/identity.ts

```diff
@@ -0,0 +1,35 @@
+import { z } from 'zod';
+import { loadConfig } from '../../storage/config.js';
+import { Signer } from '@ucanto/principal/ed25519';
+
+export const identityTool = {
+  name: 'identity',
+  description: 'Returns the DID key of the Storacha agent loaded from the private key storage config.',
+  inputSchema: z.object({}),
+  handler: async () => {
+    try {
+      const config = loadConfig();
+      
+      if (!config.privateKey) {
+        throw new Error('Private key is not defined in the storage config');
+      }
+
+      const principal = Signer.parse(config.privateKey);
```

Comment:
Ideally some sort of init for this tool would do this once, not every request. Perhaps `loadConfig` should do this and add it to the return value?